### ISSUE TITLE
Update labelflow urls

### ISSUE THREAD

**Issue by nicOwlas** [labelflow/labelflow#296]
# Tasks

⚠️ Work related to update urls with `local` workspace was done in #352 

We will add a `local` (in reference to the local workspace) into the url so that booked urls do not change when we roll-out workspaces

- `https://labelflow.ai/`: 
  - Redirect to `https://labelflow.ai/website` on first ever connection OR
  - Redirect to `https://labelflow.ai/local` for successive connections if users has clicked on `Test the app` on the website.
  - (In the future) Redirect to `https://labelflow.ai/my-workspace` , where `my-workspace` is the last workspace the user has worked on. 
- `https://labelflow.ai/website`: website always
- `https://labelflow.ai/my-workspace`: shows the datasets of the workspace `my-workspace` 
- `https://labelflow.ai/workspaces/my-workspace`: Redirect to `https://labelflow.ai/my-workspace` (this exist to allow workspaces to contain datasets that are called with special names such as `billing` or `settings`, see below)
- `https://labelflow.ai/workspaces/my-workspace/settings`: shows the workspace settings page
- `https://labelflow.ai/workspaces/my-workspace/billing`: shows the workspace billing page
- `https://labelflow.ai/my-workspace/datasets/my-dataset`: redirects to `https://labelflow.ai/my-workspace/my-dataset/images/`
- `https://labelflow.ai/my-workspace/datasets/my-dataset/images/` : image gallery page
- `https://labelflow.ai/my-workspace/datasets/my-dataset/images/b0a25998-36c4-46db-9b6b-b9620a169608`: use image id to reach the image labelling page
- `https://labelflow.ai/my-workspace/datasets/my-dataset/classes/` : list dataset classes
- `https://labelflow.ai/local`: On first connection to this page, generate the example dataset (and redirect to `https://labelflow.ai/local/example-dataset/images/[first-image-id]` once it's done)
- `https://labelflow.ai/local/example-dataset`: On first connection to this page, generate the example dataset (and redirect to `https://labelflow.ai/local/example-dataset/images/[first-image-id]` once it's done)
- `https://labelflow.ai/local/**`: `local` is the slug of the local workspace, all urls with `local` are the same as the ones with `my-workspace`, `local` is just a special name.

Urls which list all stuff:
- `https://labelflow.ai/datasets`: List all datasets that you have access to
- `https://labelflow.ai/models`: (In the future) List all models that you have access to
- `https://labelflow.ai/projects`: (In the future) List all projects that you have access to

When will be having models on top of datasets/projects
- `https://labelflow.ai/my-workspace/`: Page to be built (separate issue) which summarises the content of the work-space (`datasets`, `models`, `projects`, etc.)
- `https://labelflow.ai/my-workspace/datasets`: Show list of datasets of the workspace
- `https://labelflow.ai/my-workspace/datasets/my-dataset`: Details of my dataset
- `https://labelflow.ai/my-workspace/models`: Show list of models of the workspace
- `https://labelflow.ai/my-workspace/models/my-model`: Details of my model
- `https://labelflow.ai/my-workspace/projects`: Show list of projects of the workspace
- `https://labelflow.ai/my-workspace/projects/my-project`: Details of my project

Reserved names (to be added to workspace creation issue)
- `https://labelflow.ai/pricing`
- `https://labelflow.ai/terms`
- `https://labelflow.ai/posts`
- `https://labelflow.ai/docs`
- `https://labelflow.ai/help`
- `https://labelflow.ai/graphiql`
- `https://labelflow.ai/debug`

blocked by #352
